### ISSUE TITLE
Update to use Tesseract 5

### ISSUE THREAD

**Issue by TwoAbove** [naptha/tesseract.js-core#17]
Hey! 

Are there any plans to update core to build from tesseract 5?

I'm having trouble creating a PR because I'm having build errors
```
Checking for modules 'icu-uc;icu-i18n'
--   No package 'icu-uc' found
--   No package 'icu-i18n' found
```

I'd be happy to help if you could advise on how to fix this. Thanks!

**Comment by Balearica** [comment]
@TwoAbove I updated the .wasm version to use Tesseract 5 (for the .wasm version).  Updated files can be found at the repo link below.  The new version is significantly faster than the current version (one benchmark I ran showed ~40% reduction in runtime). 

https://github.com/naptha/tesseract.js-core/pull/18
https://github.com/Balearica/tesseract.js-core

**Comment by Balearica** [comment]
This has been merged into the master branch in #18.

**Comment by TwoAbove** [comment]
Thanks!
